src/core/components/forms/Combobox/MultiCombobox.tsx

```diff
@@ -113,17 +118,17 @@ function MultiCombobox<T extends { [key: string]: any }>(
         </OptionsWrapper>
       </div>
       {footer && footer({ close, clear: onClear })}
-    </UICombobox.Options>
+    </UIComboboxOptions>
   );
 
   return (
     <UICombobox<T[]>
       onChange={onChange}
       value={value}
       disabled={disabled}
-      multiple
+      multiple={true as any}
```

Comment:
Seems like as type bug from headlessui (might be related to https://github.com/tailwindlabs/headlessui/releases/tag/%40headlessui%2Freact%40v2.0.0#user-content-upgrading-from-v1)